src/index.ts

```diff
@@ -40,7 +46,7 @@ const run = async (): Promise<void> => {
       makeComment(prData.coverage);
     }
   } catch (error) {
-    setFailed(error.message);
+    setFailed(`${error}\ncoverage action failed to run.`);
```

Comment:
I would put the caught error message at the end like `setFailed('Coverage action failed to run: ${error.message}')`